Pipfile

```diff
@@ -19,71 +19,71 @@ alembic = "*"
 pydantic = "*"
 
 [dev-packages]
-tox = "==3.25.1"
-flake8 = "==5.0.4"
-coverage = "==6.4.4"
-black = "==22.6.0"
-isort = "==5.10.1"
-mypy = "==0.971"
-pytest = "==7.1.2"
-pretend = "==1.0.9"
-amqp = "==5.1.1"
-async-timeout = "==4.0.2"
-attrs = "==22.1.0"
-billiard = "==3.6.4.0"
-celery = "==5.2.7"
-certifi = "==2022.6.15"
-cffi = "==1.15.1"
-charset-normalizer = "==2.1.1"
-click-didyoumean = "==0.3.0"
-click-plugins = "==1.1.1"
-click-repl = "==0.2.0"
-click = "==8.1.3"
-configobj = "==5.0.6"
-cryptography = "==37.0.4"
-deprecated = "==1.2.13"
-distlib = "==0.3.5"
-dynaconf = {version = "==3.1.9", extras = ["ini"]}
-filelock = "==3.8.0"
-idna = "==3.3"
-iniconfig = "==1.1.1"
-kombu = "==5.2.4"
-mccabe = "==0.7.0"
-mypy-extensions = "==0.4.3"
-pathspec = "==0.9.0"
-platformdirs = "==2.5.2"
-pluggy = "==1.0.0"
-prompt-toolkit = "==3.0.30"
-py = "==1.11.0"
-pycodestyle = "==2.9.1"
-pycparser = "==2.21"
-pyflakes = "==2.5.0"
-pynacl = "==1.5.0"
-pyparsing = "==3.0.9"
-pytz = "==2022.2.1"
-redis = "==4.3.4"
-requests = "==2.28.1"
-securesystemslib = "==0.25.0"
-six = "==1.16.0"
-toml = "==0.10.2"
-tomli = "==2.0.1"
-tuf = "==2.0.0"
-typing-extensions = "==4.3.0"
-urllib3 = "==1.26.12"
-vine = "==5.0.0"
-virtualenv = "==20.16.3"
-watchdog = "==2.1.9"
-wcwidth = "==0.2.5"
-wrapt = "==1.14.1"
-sphinx-rtd-theme = "==1.0.0"
-sphinx = "==5.1.1"
-sphinxcontrib-applehelp = "==1.0.2"
-sphinxcontrib-devhelp = "==1.0.2"
-sphinxcontrib-htmlhelp = "==2.0.0"
-sphinxcontrib-jsmath = "==1.0.1"
-sphinxcontrib-plantuml = "==0.24"
-sphinxcontrib-qthelp = "==1.0.3"
-sphinxcontrib-serializinghtml = "==1.1.5"
+tox = "*"
+flake8 = "*"
+coverage = "*"
+black = "*"
+isort = "*"
+mypy = "*"
+pytest = "*"
+pretend = "*"
+amqp = "*"
+async-timeout = "*"
+attrs = "*"
+billiard = "*"
```

Comment:
Is the `billiard` package used? (it's a dependency of `celery`, but I don't think it's needed as a dev dep)